src/base/bittorrent/torrenthandleimpl.cpp

```diff
@@ -173,6 +173,7 @@ TorrentHandleImpl::TorrentHandleImpl(Session *session, const lt::torrent_handle
     , m_ratioLimit(params.ratioLimit)
     , m_seedingTimeLimit(params.seedingTimeLimit)
     , m_hasSeedStatus(params.hasSeedStatus)
+    , m_hasWorkingTracker(true)
```

Comment:
Using `m_hasWorkingTracker(hasWorkingTracker())` here causes updating/not contacted trackers to show as error on startup. That's why we're assuming torrent has a working tracker when added.
If anyone knows how to call that and not show error for updating/not contacted trackers then let me know.